CondCore/Utilities/scripts/getPayloadData.py

```diff
@@ -169,15 +210,17 @@ def discover():
             output(' - plot type: ', plot_type)
             single_iov = plot_method.isSingleIov()
             output(' - is single iov: ', single_iov)
-            result.setdefault(payload_type, []).append({'plot': plot, 'plugin_name': plugin_name, 'title': plot_title, 'plot_type': plot_type, 'single_iov': single_iov})
+            two_tags = plot_method.isTwoTags()
+            output(' - is Two Tags: ', two_tags)
+            result.setdefault(payload_type, []).append({'plot': plot, 'plugin_name': plugin_name, 'title': plot_title, 'plot_type': plot_type, 'single_iov': single_iov, 'two_tags': two_tags})
             output('currently discovered info: ', result)
     output('*** final output:', '')
     return json.dumps(result)
 
 def output(description, param):
     if args.verbose:
-        print('')
-        print(description, param)
+        print ''
```

Comment:
isn't this going back with the python3 changes?

Reply:
@mmusich @ggovi yes, this will fail the python3 compiler check,....

Reply:
https://github.com/cms-sw/cmssw/pull/26484 addresses the problem created by this PR.